platforms/hyperledger-besu/configuration/deploy-network.yaml

```diff
@@ -25,6 +25,17 @@
       gitops: "{{ item.gitops }}"
     loop: "{{ network['organizations'] }}"
 
+  # Create Storageclass
+  - name: Create Storage Class
+    include_role:
+      name: create/storageclass
+    vars:
+      storageclass_name: "{{ item.cloud_provider }}storageclass"
+      git_dir: "{{ item.gitops.release_dir }}"
+      org: "{{ item }}"
```

Comment:
instead of org = item, use loop_var: org

This applies for all code. All loop should have loop_var so that we don't use default "item"

Reply:
loop_var updated.